### PR TITLE
Blog Post: The Machines Can Do the Work, a Story of Kubernetes Testing, CI, and Automating the Contributor Experience

### PR DESCRIPTION
**Pull request by kbarnard10**
Adding blog post

### CODE REVIEW

File: content/en/blog/_posts/2018-08-29-kubernetes-testing-ci-automating-contributor-experience.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+With today’s [announcement from CNCF]((https://www.cncf.io/announcement/2018/08/29/cncf-receives-9-million-cloud-credit-grant-from-google)) – noting that Google Cloud has begun transferring ownership and management of the Kubernetes project’s cloud resources to CNCF community contributors –+ Google Cloud, we are excited to embark on another journey. One that allows the project infrastructure to be owned and operated by the community of contributors, following the same open governance model that has worked for the rest of the project. Sound exciting to you? Come talk to us at #sig-testing on kubernetes.slack.com.
```
Comment (neolit123) [inline content/en/blog/_posts/2018-08-29-kubernetes-testing-ci-automating-contributor-experience.md]: https://www.cncf.io/announcement/2018/08/29/cncf-receives-9-million-cloud-credit-grant-from-google
is the announcement in the works as the link is a 404?

> noting that Google Cloud has begun transferring ownership and management of the Kubernetes project’s cloud resources to CNCF community contributors –+ Google Cloud, we are excited to embark on another journey.

perhaps should be changed to:

> noting that Google has begun transferring ownership and management of the Kubernetes project’s cloud resources to a Google Cloud managed by CNCF community contributors, we are excited to embark on another journey.

Comment (BenTheElder) [reply content/en/blog/_posts/2018-08-29-kubernetes-testing-ci-automating-contributor-experience.md]: - Check the date in the URL 😉 
- We should definitely make that fix

Comment (neolit123) [reply content/en/blog/_posts/2018-08-29-kubernetes-testing-ci-automating-contributor-experience.md]: heh, it's already the 29th for me.

**Comment by zacharysarah** [comment]
Exciting and fun to read!

/approve

**Pull request merged**